src/domain/event/queryUtils.ts

```diff
@@ -72,46 +72,67 @@ export const useSimilarEventsQuery = (
   return { data, loading };
 };
 
-const useOtherEventTimesVariables = (
-  event: EventFields,
-  eventType: EventTypeId = EventTypeId.General
-) => {
+const useOtherEventTimesVariables = (event: EventFields) => {
```

Comment:
could be simplified:

```
const useOtherEventTimesVariables = (event: EventFields) => {
  const superEventId = getEventIdFromUrl(
    event.superEvent?.internalId || '',
    'event'
  );

  const variables = {
    sort: EVENT_SORT_OPTIONS.START_TIME,
    start: 'now',
    superEvent: superEventId,
    /* 
    Since LE v2 is listing general type of events 
    when no event type is given as a parameter,
    this needs a list of eventTypes
    or otherwise some events or courses are always excluded.
    */
    eventType: [EventTypeId.General, EventTypeId.Course], //event.typeId
  };

  return { superEventId, variables };
};
```

Reply:
No need for useMemos